src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/validation/YFlowValidator.java

```diff
@@ -153,6 +154,20 @@ private void checkBandwidth(YFlowRequest yFlowRequest)
         }
     }
 
+    private void checkMaxLatency(YFlowRequest yFlowRequest) throws InvalidFlowException {
+        if (yFlowRequest.getMaxLatency() == null || yFlowRequest.getMaxLatencyTier2() == null) {
+            return;
+        }
+
+        if (yFlowRequest.getMaxLatency() > yFlowRequest.getMaxLatencyTier2()) {
+            throw new InvalidFlowException(
+                    format("The maxLatency %d is higher than maxLatencyTier2 %d",
```

Comment:
after adding >= to the previous comment this message should be changed, because equal case is not good too

Reply:
done